packages/cactus-plugin-ledger-connector-iroha/src/main/json/openapi.json

```diff
@@ -173,9 +173,659 @@
+              },
+              {
+                "$ref": "#/components/schemas/GetAccountAssetTransactionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAccountAssetsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAccountDetailRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAssetInfoRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetRolePermissionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetEngineReceiptsRequestParameters"
+              }
+            ]
+          }
+        }
+      },
+      "AddAssetQuantityRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Add Asset Quantity.",
+        "type": "object",
+        "required": [
+          "assetId",
+          "amount"
+        ],
+        "properties": {
+          "assetId": {
+            "type": "string"
+          },
+          "amount": {
+            "type": "integer"
+          }
+        }
+      },
+      "AddPeerRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Add Peer.",
+        "type": "object",
+        "required": [
+          "address",
+          "peerKey"
+        ],
+        "properties": {
+          "address": {
+            "type": "string"
+          },
+          "peerKey":{
+            "type": "string"
+          },
+          "tlsCertificate":{
+            "type": "string"
+          }
+        }
+      },
+      "AddSignatoryRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Add Signatory.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "publicKey"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "publicKey": {
+            "type": "string"
+          }
+        }
+      },
+      "AppendRoleRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Append Role.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "roleName"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "roleName": {
+            "type": "string"
+          }
+        }
+      },
+      "CallEngineRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Call Engine.",
+        "type": "object",
+        "required": [
+          "caller",
+          "callee",
+          "input"
+        ],
+        "properties": {
+          "type": {
+            "type": "number",
+            "enum": [0]
+          },
```

Comment:
This looks strange for me. Is everything OK with the code? I'm not expert with OpenAPI, but other properties don't contain that fields.